AppCenter/AppCenter/MSAppCenter.m

```diff
@@ -16,13 +16,11 @@
 #import "MSCustomPropertiesPrivate.h"
 #endif
 
-// Singleton
+// Singleton.
```

Comment:
This is a comment for **definition** of variables. Should use `/** */`

Reply:
Please change the other two below. The scope for the change is remaining only on this file. We can do others with any changes of files.